client/dist/styles.css

```diff
@@ -1,10 +1,19 @@
-body {
+#title-body {
   display: flex;
   flex-direction: column;
   align-items: center;
   font-family: Arial, Helvetica, sans-serif;
-  color: #050505;
+  color: #333333;
   background-color: #f5f5f5;
+  margin: 0;
+}
+
+#yelp-header {
```

Comment:
I suggest not using so many id's since it might cause conflicts with other people's id's. Not sure if it's able to hash

Reply:
Cool